### ISSUE TITLE
False positive from no-unused-declaration for re-export

### ISSUE THREAD

**Issue by cartant** [cartant/tslint-etc#6]
`no-unused-declaration` effects a false positive for:

```ts
import { combineLatestHigherOrder } from "./combineLatestHigherOrderArray";
export { combineLatestHigherOrder };
```
